### PR TITLE
[#3007] Removed location command

### PR DESCRIPTION
**Pull request by alwx**
Targets #3007 

### Summary:

This PR removes `/location` command completely and also removes all `react-native-mapbox-gl` dependencies. It also adds a migration which removes all existing `/location` commands from `messages` table in realm database.

### CODE REVIEW

File: src/status_im/data_store/realm/schemas/account/v21/message.cljs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                          :message-type   {:type     :string
+                                           :optional true}
+                          :message-status {:type     :string
+                                           :optional true}
+                          :user-statuses  {:type       :list
+                                           :objectType :user-status}
+                          :clock-value    {:type    :int
+                                           :default 0}
+                          :show?          {:type    :bool
+                                           :default true}}})
```
Comment (janherich) [inline src/status_im/data_store/realm/schemas/account/v21/message.cljs]: It's little bit hard to see from this, but what was changes in message schema between versions V19 -> V21 ?

Comment (alwx) [reply src/status_im/data_store/realm/schemas/account/v21/message.cljs]: true, there is no need for this duplication. removed it in the last commit.

**Review by janherich — APPROVED** [review]

File: ios/StatusIm/Info.plist [hunk 2]
```diff
@@ -1,95 +1,97 @@
...
-      <string>armv7</string>
-    </array>
-    <key>UISupportedInterfaceOrientations</key>
-    <array>
-      <string>UIInterfaceOrientationPortrait</string>
-    </array>
-    <key>UIViewControllerBasedStatusBarAppearance</key>
-    <false/>
-  </dict>
+<dict>
```
Comment (yenda) [inline ios/StatusIm/Info.plist]: could you put back original indent so we can see what changed ?

**Review by yenda — APPROVED** [review]

**Review by jeluard — APPROVED** [review]
Let's remove stuff!!

**Comment by jeluard** [comment]
@alwx Please resolve conflicts

File: src/status_im/data_store/realm/schemas/account/v22/core.cljs [hunk 3]
```diff
@@ -0,0 +1,43 @@
+(ns status-im.data-store.realm.schemas.account.v22.core
```
Comment (yenda) [inline src/status_im/data_store/realm/schemas/account/v22/core.cljs]: why did you create a v22 in your rebase ? we only do one migration per version

Comment (yenda) [reply src/status_im/data_store/realm/schemas/account/v22/core.cljs]: I already had to do the rebase on my branch that is based of yours https://github.com/status-im/status-react/blob/88fa360b5f26cdfa306bbe6ad0eaa9327d8e598a/src/status_im/data_store/realm/schemas/account/v21/core.cljs

**Comment by alwx** [comment]
Android: https://i.diawi.com/hnf6kk
iOS: https://i.diawi.com/DABQcx

**Comment by yevh-berdnyk** [comment]
Checked in one-to-one and group chats cases when both users have this current app version and one of them has previous version with /location - looks good to me. Other features also verified

**Comment by yenda** [comment]
@alwx can you rebase and merge ?

**Pull request merged**